### ISSUE TITLE
"AccountSubscribe method is invalid"

### ISSUE THREAD

**Issue by kenneyLee** [gagliardetto/solana-go#192]
I am using the AccountSubscribe method to subscribe to messages from the baseVault and quoteVault accounts on Orca and Raydium, but I am not receiving any, while new SPL Transfers can already be queried on solscan.io.

The code I am running is as follows: https://github.com/viveksb007/solana-pool-monitor

**Comment by 0xvbetsun** [comment]
yeah, it looks like @kenneyLee  followed the instructions in the [old article](https://viveksb007.github.io/2022/06/amm-reserve-monitoring) with an obsolete provider URL

**Comment by ishaburov** [comment]
@taobig @0xvbetsun can you help me pls 

https://github.com/gagliardetto/solana-go/issues/165

**Comment by 0xvbetsun** [comment]
@ishaburov I'm not sure the questions that are not related to this SDK should be discussed here.

I'd recommend asking that questions in discord channels of Jupiter/solana/golang/etc.

I guess you're missing LUT in your case

**Comment by ishaburov** [comment]
> @ishaburov I'm not sure the questions that are not related to this SDK should be discussed here.
> 
> I'd recommend asking that question in discord channel of Jupiter/solana/golang/etc.
> 
> I guess you're missing LUT in your case

Thanks. From Api i got instructions and addressLookupTableAccounts how i can build it and insert to transaction?
